specification.markdown

```diff
@@ -749,6 +749,56 @@ type: Unsigned Integer
 
 description: The minimum `DocType` version an `EBML Reader` has to support to read this `EBML Document`. The value of the `DocTypeReadVersion Element` MUST be less than or equal to the value of the `DocTypeVersion Element`.
 
+### DocTypeExtension Element
+
+name: `DocTypeExtension`
+
+path: `0*(\EBML\DocTypeExtension)`
+
+id `0x4281`
+
+minOccurs: 0
+
+type: `Master Element`
+
+description: A `DocType` extension adds extra `Elements` to the main `DocType`+`DocTypeVersion` tuple it's attached to. An `EBML Reader` MAY understand these extra `Elements`. It MAY be used to iterate between experimental `Elements` before they are integrated in a regular `DocTypeVersion`. Reading one `DocTypeExtension` version of a `DocType`+`DocTypeVersion` tuple doesn't imply one should be able to read upper values of this `DocTypeExtension`.
```

Comment:
> It MAY be used to…

This "it" can be misunderstood to refer to "An `EBML Reader`" from the previous sentence. Changing it to "A `DocTypeExtension` MAY be used…" would remove the ambiguity.